src/target_triple.rs

```diff
@@ -53,40 +53,37 @@ impl TargetTriple {
     }
 
     pub fn from_component(component: &str) -> Result<Self> {
```

Comment:
can we add a doc comment. feel free to tweak the below 

```
/// Creates a target triple for the given component.
/// 
/// For components distributed by forc, uses the format "{os}_{architecture}".
/// For other components, uses the format "{architecture}-{vendor}-{os}".
```

Reply:
Cool, done.